data/editor.config

```diff
@@ -991,6 +991,9 @@
       <type id="attraction-animal">
         <include field="name" />
       </type>
+      <type id="tourism-camp_pitch" group="accomodation">
```

Comment:
What's the value of adding individual pitches as POIs?

Reply:
As discussed below -- my goal was just to have these clickable / searchable for the purpose of navigating. I'm able to be convinced to have them simply displayable instead, pending resolution of zoom level questions and not overcrowding.